sunpy/map/mapsequence.py

```diff
@@ -81,6 +81,16 @@ def __len__(self):
         """Return the number of maps in a mapsequence."""
         return len(self.maps)
 
+    def __repr__(self):
+        names = []
+        for m in self.maps:
+            name = m.nickname + 'Map'
```

Comment:
Just use the nickname by itself.  What you've done won't be correct for those Maps that provide more informative nicknames.

Alternatively, use the actual name of the class (i.e., `m.__class__.__name__`).